borg/archiver.py

```diff
@@ -317,21 +310,22 @@ def do_delete(self, args):
             repository.commit()
             cache.commit()
             if args.stats:
-                stats.print_('Deleted data:', cache)
+                logger.info(stats.print_('Deleted data:', cache))
         else:
             if not args.cache_only:
-                print("You requested to completely DELETE the repository *including* all archives it contains:")
+                print("You requested to completely DELETE the repository *including* all archives it contains:", file=sys.stderr)
```

Comment:
print to channel 2 = print to stderr, less typing.

i don't want to call it print_err because it doesn't need to be an error we output.